symplyphysics/conditions/electricity/maxwell_equations/derivative_of_magnetic_induction_in_time_is_rotor_of_electric_intensity.py

```diff
@@ -0,0 +1,30 @@
+from sympy import Eq, Derivative, Matrix
+from sympy.core import Equality
+from symplyphysics import (units, Symbol)
+from symplyphysics.core.fields.operators import curl_operator
+from symplyphysics.core.fields.vector_field import VectorField
+
+
+## Description
+## Faraday's law of induction states that a change in magnetic induction generates a vortex electric field.
+## This law is valid for any magnetic field that changes over time.
+
+## Law is: curl(E(r, t)) = -d(B(r, t)) / dt, where
```

Comment:
Could you find some reference values for E and B to test this condition?

Reply:
> Could you find some reference values for E and B to test this condition?

Yes, I will try to find the fields E and B that satisfy the law.